### ISSUE TITLE
[Lotus Gateway] - Filecoin.StateNetworkName is not a part of Lotus Gateway

### ISSUE THREAD

**Issue by mur-me** [filecoin-project/lotus#10402]
### Checklist

- [X] This is **not** a security-related bug/issue. If it is, please follow please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
- [X] This is **not** a question or a support request. If you have any lotus related questions, please ask in the [lotus forum](https://github.com/filecoin-project/lotus/discussions).
- [X] This is **not** a new feature request. If it is, please file a [feature request](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Ffeature&template=feature_request.yml) instead.
- [X] This is **not** an enhancement request. If it is, please file a [improvement suggestion](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Fenhancement&template=enhancement.yml) instead.
- [X] I **have** searched on the [issue tracker](https://github.com/filecoin-project/lotus/issues) and the [lotus forum](https://github.com/filecoin-project/lotus/discussions), and there is no existing related issue or discussion.
- [X] I am running the [`Latest release`](https://github.com/filecoin-project/lotus/releases), or the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [X] I did not make any code changes to lotus.

### Lotus component

- [ ] lotus daemon - chain sync
- [ ] lotus miner - mining and block production
- [ ] lotus miner/worker - sealing
- [ ] lotus miner - proving(WindowPoSt)
- [ ] lotus miner/market - storage deal
- [ ] lotus miner/market - retrieval deal
- [ ] lotus miner/market - data transfer
- [ ] lotus client
- [X] lotus JSON-RPC API
- [ ] lotus message management (mpool)
- [ ] Other

### Lotus Version

```text
Daemon:  1.20.0+mainnet+git.50a73ba+api1.5.0
Local: lotus version 1.20.0+mainnet+git.50a73ba
```

### Describe the Bug

Currently Lotus Gateway doesn't support `Filecoin.StateNetworkName`. Since we at Glif now serve our public endpoints using Lotus Gateway, some applications dependent on this method stopped working.

We'd like the method to be implemented as part of Lotus Gateway.

### Logging Information

```text
N/A
```

### Repo Steps

1. Run 
`curl --data '{"jsonrpc": "2.0", "id":1, "method": "Filecoin.StateNetworkName", "params": [] }' -H 'Content-Type:application/json' https://api.node.glif.io/`

**Comment by jennijuju** [comment]
Arsenii from protofire will give this a pass

**Comment by rjan90** [comment]
Closed by #10420
